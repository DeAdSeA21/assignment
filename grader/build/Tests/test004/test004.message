Output line 14:
  <i>expected: </i><span class='expected'>13</span>\t958.63
  <i>observed: </i>-<span class='observed'>-</span>- end of output ---
