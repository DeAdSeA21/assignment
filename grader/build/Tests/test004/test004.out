Starting loan balance: Annual interest rate: Monthly payment: 
Month	Balance
1	996.83
2	993.66
3	990.49
4	987.32
5	984.14
6	980.96
7	977.78
8	974.59
9	971.40
10	968.21
11	965.02
12	961.83
